CMakeLists.txt

```diff
@@ -459,16 +462,11 @@ endif()
 
 # clean up the requirements.txt files from 3rd party project folder to suppress the code security false alarms
 file(GLOB_RECURSE NO_USE_FILES ${CMAKE_BINARY_DIR}/_deps/*requirements.txt)
-message(STATUS "Found the follow requirements.txt: ${NO_USE_FILES}")
+message(STATUS "Found the following requirements.txt: ${NO_USE_FILES}")
 
 foreach(nf ${NO_USE_FILES})
-  file(TO_NATIVE_PATH ${nf} nf_native)
-
-  if(CMAKE_SYSTEM_NAME MATCHES "Windows")
-    execute_process(COMMAND cmd /c "del ${nf_native}")
-  else()
-    execute_process(COMMAND bash -c "rm ${nf_native}")
-  endif()
+    file(TO_NATIVE_PATH ${nf} nf_native)
```

Comment:
Looks better, is to_native_path still needed then?

Reply:
Without TO_NATIVE_PATH the path always has `/` in it. That works as Windows allows both `\\` and `/` as a path separator, so strictly speaking it isn't necessary.